packages/tocco-ui/src/EditableValue/typeEditors/DateTimeEdit.spec.js

```diff
@@ -12,7 +19,9 @@ describe('tocco-ui', () => {
       describe('DateTimeEdit ', () => {
         test('should render an instance of DateAbstract', () => {
           const wrapper = intlEnzyme.mountWithIntl(
-            <DateTimeEdit onChange={EMPTY_FUNC}/>
+            <ThemeProvider theme={theme}>
```

Comment:
what changed that all these <ThemeProvider>s re needed?

Reply:
Function generateDisabledShade in test "tocco-ui › EditableValue › typeEditors › DateTimeEdit  › should render an instance of DateAbstract" fails  without ThemeProvider.

Reply:
I agree and have already changed that. Function generateDisabledShade base upon function chadeColor which has a fallback color defined. Therefore generateDisabledShade is safe and it is also possible to remove every ThemeProvider from EditableValue tests.